### DISCUSSION TITLE
No update step for 'RV_DEPS_OPENEXR'... 'patch' is not recognized as an internal or external command,

### DISCUSSION THREAD

**Discussion by nebukadhezer** [AcademySoftwareFoundation/OpenRV#295] [Build Issues] [answered]
No update step for 'RV_DEPS_OPENEXR'
         Performing patch step for 'RV_DEPS_OPENEXR'
         'patch' is not recognized as an internal or external command,
         operable program or batch file.

I am on windows 11, I do have another install of vcpkg... could this be related ?
Anyone has seen this too ?

**Comment by rogernelson** [comment, marked as answer]
I think you are just missing the patch binary. I don't have a windows box in front of me, but doing something like `pacman -S patch` should probably do the trick.

**Reply by nebukadhezer** [reply]
God, I do feel a bit stupid right now. Thanks that was it builds fine now.

**Reply by nebukadhezer** [reply]
https://github.com/AcademySoftwareFoundation/OpenRV/blob/main/docs/build_system/config_windows.md#5-install-required-msys2-pacman-packages

Maybe this can be added here.

**Reply by rogernelson** [reply]
You're right, that's definitely an oversight that it's not there. I will add it.

**Reply by rogernelson** [reply]
I'm glad it's working for you. The Windows build can be tricky so good job getting it to build!

**Reply by nebukadhezer** [reply]
To me the documentation is good. The biggest hurdle is to get all downloaded and installed I guess...  the ACLOCAL_PATH is about the only thing that is not in the readme that I have to set/export, to make it work on win11.
Thanks again for your help.

**Reply by rogernelson** [reply]
Thanks for reporting it.  I opened a PR here:
https://github.com/AcademySoftwareFoundation/OpenRV/pull/297
